extension/parquet/parquet_extension.cpp

```diff
@@ -70,8 +70,8 @@ struct ParquetReadBindData : public TableFunctionData {
 	// These come from the initial_reader, but need to be stored in case the initial_reader is removed by a filter
 	idx_t initial_file_cardinality;
 	idx_t initial_file_row_groups;
+	idx_t explicit_cardinality; // can be set to inject exterior knowledge on total cardinality (e.g. from a data lake)
```

Comment:
Can we explicitly intitialize this with 0? It seems this is currently uninitialized.

Reply:
done, sorry I thought the whole struct was initialized 0